plugins/ruby-foreman-remote-execution/debian/install

```diff
@@ -1,4 +1,4 @@
 foreman_remote_execution.rb           usr/share/foreman/bundler.d
 cache                                 usr/share/foreman/vendor
-foreman_remote_execution              var/lib/foreman/public/assets
+assets                                var/lib/foreman/public
```

Comment:
It should be OK, but you also need to update the `rules` so it copies the whole assets directory (which is beneath GEM_PATH, so will only be this plugin's assets).  We'll have to check the resulting build contents.